### PR TITLE
Upgraded Skiko version to 0.8.18

### PR DESCRIPTION
**Pull request by mazunin-v-jb**
Skiko changelog: https://github.com/JetBrains/skiko/compare/v0.8.17...v0.8.18

Fixes - https://youtrack.jetbrains.com/issue/CMP-6879

### Fixes - Multiple Platforms
- Fix Skia paragraph caching performance degradation

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -68,7 +68,7 @@ protobuf = "3.22.3"
 paparazzi = "1.0.0"
 paparazziNative = "2022.1.1-canary-f5f9f71"
 shadow = "8.1.1"
-skiko = "0.8.17"
```
Comment (igordmn) [inline gradle/libs.versions.toml:71]: We need to fix tests, adding more relaxed checks for values

**Review by igordmn — APPROVED** [review]

File: gradle/libs.versions.toml (hunk 1 above)
Comment (igordmn) [reply gradle/libs.versions.toml:71]: Still has failed tests

Comment (mazunin-v-jb) [reply gradle/libs.versions.toml:71]: accidentally made tolerance too small, now should be good

**Review by igordmn — APPROVED** [review]

**Comment by igordmn** [comment]
Changed:
```
### Fixes - Multiple Platforms
Fix Skia paragraph caching performance degradation
```
to the correct format:
```
## Release Notes
### Fixes - Multiple Platforms
- Fix Skia paragraph caching performance degradation
```

**Pull request merged**